Sources/DolbyIOUIKit/Theme/Asset/Theme.swift

```diff
@@ -3,38 +3,158 @@
 //  
 import SwiftUI
 
-public protocol ThemeProviding {
-    associatedtype _ColorAsset: ColorAssetable
-    associatedtype _FontAsset: FontAssetable
-    associatedtype _ImageAsset: ImageAssetable
-
-    subscript(colorAsset: _ColorAsset) -> Color? { get }
-    subscript(fontAsset: _FontAsset) -> Font { get }
-    subscript(imageAsset: _ImageAsset) -> Image { get }
-}
+public protocol Theme {
+    // Material theme color definitions
```

Comment:
Not sure if we shall mention the Material theme here. It is Android. Open to this.

Reply:
It is not specific to Android, other platforms also use it

Reply:
Looking at its implementation in `DefaultTheme` they are all set to use `UIColor.neonPurple400` at the moment. Is that as expected?

Reply:
Will update it in future, set it to default `UIColor.neonPurple40` as we don't have all the definitions in the design yet